.github/workflows/deno.yml

```diff
@@ -2,9 +2,7 @@ name: Deno
 
 on:
   push:
-    branches: [main]
```

Comment:
Without this line, if you make a branch in this repo and then make a PR, CI will run twice.

Reply:
yeah. But with it creating a branch without a PR it will not run CI at all which is annoying too…